ios/ci_scripts/ci_post_clone.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Fail this script if any subcommand fails.
+set -e
+
+# The default execution directory of this script is the ci_scripts directory.
+cd $CI_PRIMARY_REPOSITORY_PATH # change working directory to the root of your cloned repo.
+
+# Install Flutter using git.
+git clone https://github.com/flutter/flutter.git --depth 1 -b stable $HOME/flutter
+export PATH="$PATH:$HOME/flutter/bin"
+
+# Install Flutter artifacts for iOS (--ios), or macOS (--macos) platforms.
+flutter precache --ios
+
+# Install Flutter dependencies.
+flutter pub get
```

Comment:
In the future, we should probably use hermit here. Hopefully, we just need a ```sh
./bin/activate-hermit
just get
```

Reply:
oh yes might try that on another branch - to get used to to xcode build env